### PR TITLE
Letter spacing CSS support added

### PR DESCRIPTION
**Pull request by enchev**
Implemented #1757

### CODE REVIEW

**Comment by enchev** [comment]
run ci

**Comment by dtopuzov** [comment]
run ci

**Comment by dtopuzov** [comment]
run ci

File: ui/text-base/text-base-styler.android.ts [hunk 1]
```diff
@@ -108,6 +108,19 @@ export class TextBaseStyler implements style.Styler {
         utils.ad.setWhiteSpace(view._nativeView, enums.WhiteSpace.normal);
     }
 
+    // letter-spacing
```
Comment (hshristov) [inline ui/text-base/text-base-styler.android.ts]: These are API 21 methods so we should check if methods is available at runtime before using them.

**Comment by hshristov** [comment]
:+1:  After check for API 21

**Comment by enchev** [comment]
run ci

**Pull request merged**